.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+<!--
+Thanks for contributing!
+-->
+
+Please complete the following before merging:
+- [ ] Bump spec version and last modified date.
+- [ ] Update changelog.
+- [ ] Make sure there are generated JSON files from the YAML test files.
+- [ ] Test changes in at least one language driver.
```

Comment:
I'd add something like: "Test these changes against all server versions and topologies (including standalone, replica set, sharded clusters, and serverless).

We often merge changes after only testing on one server version or only one topology type which leads to test churn.